### ISSUE TITLE
does not works with Adminer Editor 4.6.3

### ISSUE THREAD

**Issue by DiegoJArg** [LinkedList/Adminer-Allowed-Prefixes#1]
Hi.
I realised that this plugin may be useful for me by using a user-side interface with Adminer Editor.
However it doesnt works.

This line does not returns any value when executed with Editor.
$database = Adminer::database();

**Comment by DiegoJArg** [comment]
By hardcoding the prefix, it works
$this->prefixes = "__"; //$prefixes[$server][$database];

However, it does not filters the main page of the server

**Comment by LinkedList** [comment]
Hi @jacobidiego , I have not used this plugin for a long time, however I'll try to update it if it is useful for you :+1: 

Feel free to submit a PR though, if you fix it before me :slightly_smiling_face:

**Comment by DiegoJArg** [comment]
Hi LinkedList, 
which plugins would you currently use to filter tables out?

Also, if you are going to edit this file, please disable the filtering for the root user, allowing the root to see everything, and filter the list of tables when it is shown in the "database view" or the "Server view" if using Editor.

**Comment by LinkedList** [comment]
@jacobidiego I've just updated the plugin to work with latest version 4.6.3.

However the functionality to show different tables to different users is not something this plugin was writter for, so I'll leave that to you. If you would implement this somehow with easy configuration I would consider adding it to the main branch. Closing this for now, open another Issue if you find something and thanks for the report. :+1:
